Sources/StreamChatUI/CommonViews/InputChatMessageView/InputChatMessageView.swift

```diff
@@ -96,12 +115,35 @@ open class _InputChatMessageView<ExtraData: ExtraDataTypes>: _View, ComponentsPr
         NSLayoutConstraint.activate([
             clearButton.heightAnchor.pin(equalToConstant: 24),
             clearButton.widthAnchor.pin(equalTo: clearButton.heightAnchor, multiplier: 1),
-            imageAttachmentsView.heightAnchor.pin(equalToConstant: 120)
+            imageAttachmentsCollectionView.heightAnchor.pin(equalToConstant: 120)
         ])
     }
 
-    public func setSlashCommandViews(hidden: Bool) {
-        commandLabelView.isHidden = hidden
-        clearButton.isHidden = hidden
+    override open func updateContent() {
+        super.updateContent()
+
+        guard let content = self.content else { return }
+
+        if let quotingMessage = content.quotingMessage {
```

Comment:
super nit: Why not quoted message?

Reply:
The reason is that the message is not yet quoted, we are "quoting" or "replying" at the moment 😄 But maybe I'm wrong, naming is always subjective 😛